.github/workflows/build_changelog.yml

```
@@ -17,6 +17,9 @@ on:
     branches:
       - develop
 
+permissions:
+  contents: read
+
```

```suggestion
  # job-level permissions block has usage explanation
  contents: write
  pull-requests: write
```

Correct top-level permission to prevent failing job. Jobs cannot override top-level permission for good reasons. Scopes not specified at job-level will be overridden to `None`